Resources/Private/JavaScript/Host/Containers/ContextBar/index.js

```diff
@@ -25,27 +27,34 @@ export default class ContextBar extends Component {
     };
 
     render() {
-        const {isFringedLeft, isFringedRight, isFullScreen} = this.props;
+        const {previewUrl, isFringedLeft, isFringedRight, isFullScreen} = this.props;
         const classNames = mergeClassNames({
             [style.contextBar]: true,
             [style['contextBar--isFringeLeft']]: isFringedLeft,
             [style['contextBar--isFringeRight']]: isFringedRight,
             [style['contextBar--isHidden']]: isFullScreen
         });
+        const previewButtonClassNames = mergeClassNames({
+            [style.contextBar__buttonLink]: true,
+            [style['contextBar__buttonLink--isDisabled']]: !previewUrl
+        });
 
         return (
             <div className={classNames}>
                 <DimensionSwitcher />
 
                 <div className={style.contextBar__rightHandedActions}>
-                    <IconButton icon="external-link" onClick={this.onClickOpenInNewTab.bind(this)} />
+                    {previewUrl ?
```

Comment:
I would personally just add an anchor without an href attribute if the previewUrl was not passed. Also note https://mathiasbynens.github.io/rel-noopener/